yang/vyatta-protocols-frr-isis-v1.yang

```diff
@@ -0,0 +1,822 @@
+          configd:help "Delay used while in SHORT_WAIT state";
+          description "Delay used while in SHORT_WAIT state";
+          key "short-val";
+          leaf short-val {
+            type uint16 {
+              range "0..60000";
+            }
+            units "msec";
+            mandatory true;
+          }
+          list long-delay {
+            configd:help "Delay used while in LONG_WAIT state";
+            description "Delay used while in LONG_WAIT state";
+            key "long-val";
+            leaf long-val {
+              type uint16 {
+                range "0..60000";
+              }
+              units "msec";
+              mandatory true;
+            }
+            list holddown {
+              configd:help "Time with no received IGP events before considering IGP stable";
+              description "Time with no received IGP events before considering IGP stable";
+              key "hold-val";
+              leaf hold-val {
+                type uint16 {
+                  range "0..60000";
+                }
+                units "msec";
+                mandatory true;
+              }
+              list time-to-learn {
+                configd:help "Max duration needed to learn all the events related to a single failure";
+                description "Max duration needed to learn all the events related to a single failure";
+                key "t2l-val";
+                leaf t2l-val {
+                  type uint16 {
+                    range "0..60000";
+                  }
+                  units "msec";
+                  mandatory true;
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+
+    container spf-interval {
+      configd:help "Minimum interval between SPF calculations";
+      description "Minimum interval between SPF calculations";
+      leaf level-1 {
+        type uint16 {
+          range "1..120";
+        }
+        units "seconds";
+        configd:help "Minimum time between consecutive level-1 SPFs.";
+        description "Minimum time between consecutive level-1 SPFs.";
+      }
+      leaf level-2 {
+        type uint16 {
+          range "1..120";
+        }
+        units "seconds";
+        configd:help "Minimum time between consecutive level-2 SPFs.";
+        description "Minimum time between consecutive level-2 SPFs.";
+      }
+    }
+
+    leaf is-type {
+      type level;
+      configd:help "IS Level for this routing process (OSI only)";
+    }
+
+    leaf metric-style {
+      type metric-style-type;
+      description "Define the style of TLVs metric supported.";
+      configd:help "Define the style of TLVs metric supported.";
+    }
+
+    leaf lsp-mtu {
+      type uint16 {
+        range "128..4352";
+      }
+      configd:help "MTU of an LSP.";
+      description "MTU of an LSP.";
+    }
+  }
+
+	grouping redistribute-protocol {
+		leaf kernel {
+      type level;
+      must ". != \"isis\"";
```

Comment:
These must statements in this grouping can be removed, since the `isis` is not defined by the `level` enumeration.

Reply:
This is Resolved.